hooks/post/10-setup_provisioning.rb

```diff
@@ -52,7 +52,19 @@ def run_rails
     system("echo ':restrict_registered_smart_proxies: false' >> #{DEPLOYMENT_DIR}/foreman/config/settings.yaml")
 
     run_rails
-    say "Rails is running in PID: #{rails_pid}"
+    say "Waiting up to 60 seconds for 200 response code"
```

Comment:
How about increasing timeout to 5 minutes?
I've seen times when I wait ~2 minutes for /users/login to render.